src/react-integration/provider/ExternalCacheProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode, useEffect, useState } from 'react';
```

Comment:
this file is the only *real* changed in code. this is mostly tests and docs as it was already built to be easy redux integration.